components/match2/wikis/valorant/match_group_input_custom.lua

```diff
@@ -89,6 +89,7 @@ function CustomMatchGroupInput.extractMaps(match, opponents)
 		local winnerInput = map.winner --[[@as string?]]
 
 		map.opponents = MapFunctions.getParticipants(map, opponents)
+		map.participants = nil
```

Comment:
maybe add a comment on why this is needed

Reply:
added